Testscripts/Linux/VERIFY-LIS-MODULES-VERSION.sh

```diff
@@ -33,16 +33,17 @@ function check_version_greater_equal() { test "$(echo "$@" | tr " " "\n" | sort
 # Check if vmbus string is recorded in dmesg
 hv_string=$(dmesg | grep "Vmbus version:")
 if [[ ( $hv_string == "" ) || ! ( $hv_string == *"hv_vmbus:"*"Vmbus version:"* ) ]]; then
-    LogErr "Error! Could not find the VMBus protocol string in dmesg."
+    LogErr "Could not find the VMBus protocol string in dmesg. Test stopped here."
     SetTestStateAborted
     exit 0
 fi
 
 skip_modules=()
 config_path="/boot/config-$(uname -r)"
 if [[ $(detect_linux_distribution) == clear-linux-os ]]; then
-    config_path="/usr/lib/kernel/config-$(uname -r)"
+	config_path="/usr/lib/kernel/config-$(uname -r)"
```

Comment:
indent

Reply:
fixed in 78f5c4a94ea94a2156d5d01eebe191584c64fcdf